pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -72,8 +73,14 @@ func NewPredictor(client client.Client, clientset kubernetes.Interface, scheme *
 func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, error) {
 	var container *v1.Container
 	var podSpec v1.PodSpec
+	var workerPodSpec v1.PodSpec
+	var workerObjectMeta metav1.ObjectMeta
+	var multiNodeEnabled bool
+	var sRuntime v1alpha1.ServingRuntimeSpec
 	var sRuntimeLabels map[string]string
 	var sRuntimeAnnotations map[string]string
+	var sRuntimeWorkerLabels map[string]string
+	var sRuntimeWorkerAnnotations map[string]string
```

Comment:
Can't be the annotations part of the metadata?

Reply:
ObjectMeta*

Reply:
It is eventually added to `ObjectMeta` . please check in Line #417